src/emulator/downloadableEmulators.ts

```diff
@@ -51,16 +51,18 @@ const DownloadDetails: { [s in DownloadableEmulators]: EmulatorDownloadDetails }
     },
   },
   gui: {
-    version: "0.0.1",
-    downloadPath: path.join(CACHE_DIR, "gui-v0.0.1.zip"),
-    unzipDir: path.join(CACHE_DIR, "gui-v0.0.1"),
-    binaryPath: path.join(CACHE_DIR, "gui-v0.0.1", `server.bundle.js`),
+    version: "0.0.4",
+    downloadPath: path.join(CACHE_DIR, "gui-v0.0.4.zip"),
+    unzipDir: path.join(CACHE_DIR, "gui-v0.0.4"),
+    binaryPath: path.join(CACHE_DIR, "gui-v0.0.4", `server.bundle.js`),
     opts: {
       cacheDir: CACHE_DIR,
-      remoteUrl: "https://storage.googleapis.com/firebase-preview-drop/emulator/gui-v0.0.1.zip",
-      expectedSize: 1204964,
-      expectedChecksum: "52847b962bb66de639487d96a07a69d3",
+      remoteUrl: "https://storage.googleapis.com/firebase-preview-drop/emulator/gui-v0.0.4.zip",
```

Comment:
Is the plan to continually overwrite this version?  If so should we use a meaningless version like `9.9.9-SNAPSHOT`

Also we could use a convention that `-SNAPSHOT` builds are not validated for checksum (rather than add a new boolean field).

Reply:
we're planning on moving to a single URL for EAP which we just overwrite (hence the checksum / size flags I added), I have no opinions on anything beyond that.

Reply:
I'm down for `-SNAPSHOT` and the convention.